src/dbg/commands/cmd-searching.cpp

```diff
@@ -754,6 +754,7 @@ static bool cbModCallFind(Zydis* disasm, BASIC_INSTRUCTION_INFO* basicinfo, REFI
     {
         GuiReferenceInitialize(refinfo->name);
         GuiReferenceAddColumn(2 * sizeof(duint), GuiTranslateText(QT_TRANSLATE_NOOP("DBG", "Address")));
+        GuiReferenceAddColumn(0 * sizeof(duint), GuiTranslateText(QT_TRANSLATE_NOOP("DBG", "Module")));

```

Comment:
The value `0` is a magic number used only for the last column to use 'the rest' of the space. Some default that works for 'most' modules would be better.